docs/api.md

```diff
@@ -178,12 +180,24 @@ class DummyCompositeAlgo(tools.CompositeAlgo):
         predictions = 0
         return predictions
 
-    def load_model(self, path):
+    def _load_model(self, path):
         return json.load(path)
 
-    def save_model(self, model, path):
+    def _save_model(self, model, path):
         json.dump(model, path)
```

Comment:
Since we're trying to illustrate the most basic implementation, maybe we shouldn't mutualize this little piece of code : it seems to indicate that load_head/load_trunk are supposed to do the same thing.

Reply:
Good point, there is no reason to be lazy here!

Reply:
Done